vagrant-post-script/orchestrate_docker.sh

```diff
@@ -1,8 +1,4 @@
 #!/bin/bash
 
 cd oms-docker/docker
-docker-compose up -d
-echo "############# ok until here"
-docker-compose logs -f omscore-bootstrap
-docker-compose logs -f omsevents-bootstrap
-echo "############# done"
+make bootstrap
```

Comment:
True, but again it standardises for mac and windows - also it could model the future infrastructure (currently it is an ubuntu box which is same version that is going to be on titan), and the main difference between hosts is the docker daemon which does influence a bit! For the company I work for I created the same setup (pick between vagrant+ or just docker) and pure docker didn't run smoothly

And of course the possibility to create a cluster on some machines with minimal setup

(Finally true about Make, but it is present by default on most distros, even arch :) )